Controller/RUL2/11_Rail/Sec11s_RRW_Height-Transition.txt

```diff
@@ -0,0 +1,515 @@
+	0x5d740900,0,0,0x03002200,2,0=0x5d740d00,0,0,0x5d740b00,2,0
+	0x5d740900,2,1,0x03002200,0,1=0x5d740d00,2,1,0x5d740b00,0,1
+	0x5d740d00,3,0,0x5d540800,2,0=0x5d740d00,3,0,0x5d740c00,3,0
+	0x5d740d00,1,1,0x5d540800,0,1=0x5d740d00,1,1,0x5d740c00,1,1
+	0x5d740d00,3,0,0x5d740800,3,0=0x5d740d00,3,0,0x5d740c00,3,0
+	0x5d740d00,1,1,0x5d740800,1,1=0x5d740d00,1,1,0x5d740c00,1,1
+	0x5d740b00,3,0,0x03031600,1,0=0x5d740b00,3,0,0x5d740a00,3,0
+	0x5d740b00,1,1,0x03031600,3,1=0x5d740b00,1,1,0x5d740a00,1,1
+;diagonal OST
+0x5d7f010e,0,0,0x5d540300,2,1=0x5d7f0100,0,0,0x5d740100,2,0
+0x5d7f0100,0,0,0x5d540300,2,1=0x5d7f0100,0,0,0x5d740100,2,0
+0x5d7f020e,2,0,0x5d540300,2,1=0x5d7f0200,2,0,0x5d540100,2,0
+0x5d7f0200,2,0,0x5d540300,2,1=0x5d7f0200,2,0,0x5d540100,2,0
+0x5d7f010e,2,1,0x5d740100,0,1=0x5d7f0100,2,1,0x5d740100,1,0
+	;single tile viaduct side
+	0x5d7f010e,0,0,0x5f33fc00,0,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f010e,0,0,0x5f33fc00,1,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f010e,0,0,0x5f33fc00,2,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f010e,0,0,0x5f33fc00,3,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f0100,0,0,0x5f33fc00,0,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f0100,0,0,0x5f33fc00,1,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f0100,0,0,0x5f33fc00,2,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f0100,0,0,0x5f33fc00,3,0=0x5d7f0100,0,0,0x5d740100,2,0
+	0x5d7f010e,2,1,0x5f33fc00,0,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f010e,2,1,0x5f33fc00,1,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f010e,2,1,0x5f33fc00,2,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f010e,2,1,0x5f33fc00,3,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f0100,2,1,0x5f33fc00,0,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f0100,2,1,0x5f33fc00,1,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f0100,2,1,0x5f33fc00,2,0=0x5d7f0100,2,1,0x5d740100,1,0
+	0x5d7f0100,2,1,0x5f33fc00,3,0=0x5d7f0100,2,1,0x5d740100,1,0
+	;single tile slope side
+	0x5d7f020e,2,0,0x5f33fc00,0,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f020e,2,0,0x5f33fc00,1,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f020e,2,0,0x5f33fc00,2,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f020e,2,0,0x5f33fc00,3,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f0200,2,0,0x5f33fc00,0,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f0200,2,0,0x5f33fc00,1,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f0200,2,0,0x5f33fc00,2,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f0200,2,0,0x5f33fc00,3,0=0x5d7f0200,2,0,0x5d540100,2,0
+	0x5d7f020e,0,1,0x5f33fc00,0,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f020e,0,1,0x5f33fc00,1,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f020e,0,1,0x5f33fc00,2,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f020e,0,1,0x5f33fc00,3,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f0200,0,1,0x5f33fc00,0,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f0200,0,1,0x5f33fc00,1,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f0200,0,1,0x5f33fc00,2,0=0x5d7f0200,0,1,0x5d540100,1,0
+	0x5d7f0200,0,1,0x5f33fc00,3,0=0x5d7f0200,0,1,0x5d540100,1,0
+;conversion of flipped diagonal
+0x5d7f0100,2,1,0x5d740100,0,1=0x5d7f0100,2,1,0x5d740100,1,0
```

Comment:
Are you sure this works? This "anti-mirroring" has the same effect as

    0x5d7f0100,0,0,0x5d740100,2,0=0x5d7f0100,0,0,0x5d740100,3,1

Reply:
I removed the 'anti-mirroring' rules.  With these changes, and the changes to the RRW rule generator, there should be no more instances of `5d640100 / 5d740100` in a mirrored configuration.